basic/exercise7/script7.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+function getNumberOfDaysUntilNextChristmas(date) {
+    var oneDayInMiliseconds = 1000 * 60 * 60 * 24;
+    var myDate = new Date;
+    var christmasDate = new Date(myDate.getFullYear(), 11, 25);
+    if (myDate.getMonth() === 11 && myDate.getDate() > 25) {
+        christmasDate.setFullYear(christmasDate.getFullYear() + 1)
+    };
+    var timeUntilChristmas = Math.ceil((christmasDate.getTime() - myDate.getTime()) / (oneDayInMiliseconds));
+    var dayUntilChristmas = `There are ${timeUntilChristmas} days to Christmas.`;
```

Comment:
Why do you return this string?
Funcion is called get number of days, not get string containing somewhere information about number of days.